synapse/tools/storm.py

```diff
@@ -166,6 +167,52 @@ async def runCmdOpts(self, opts):
         except s_exc.SynErr as e:
             self.printf(e.errinfo.get('mesg', str(e)))
 
+class ExportCmd(StormCliCmd):
+    '''
+    Export the results of a storm query into a nodes file.
+
+    Example:
+
+        // Export nodes to a file
+        !export dnsa.nodes { inet:fqdn#mynodes -> inet:dns:a }
+
+        // Export nodes to a file and only include specific tags
+        !export fqdn.nodes { inet:fqdn#mynodes } --includetags footag
+    '''
+
+    _cmd_name = '!export'
+
+    def getArgParser(self):
+        pars = StormCliCmd.getArgParser(self)
+        pars.add_argument('filepath', help='The file path to save the export to.')
+        pars.add_argument('query', help='The Storm query to export nodes from.')
+        pars.add_argument('--includetags', nargs='*', help='Only include the specified tags in output.')
```

Comment:
Lets tweak this to `--include-tags` and also add a `--no-tags` option (without arguments) to make it obvious that you can do that too :)

Reply:
updated